gnoi/cert/manager.go

```diff
@@ -52,11 +59,20 @@ type Manager struct {
 }
 
 // NewManager returns a Manager.
-func NewManager(privateKey crypto.PrivateKey) *Manager {
+func NewManager(settings *Settings) *Manager {
+	if settings == nil {
```

Comment:
No need to have this check, we should fail hard if this happens. If we want to have an empty Manager{} we can always create it without calling this function.